mudcontent.ttl

```diff
@@ -0,0 +1,25 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix mud: <https://raw.githubusercontent.com/Multi-User-Domain/vocab/main/mud.ttl#> .
+@prefix mudcontent: <https://raw.githubusercontent.com/Multi-User-Domain/vocab/main/mudcontent.ttl#> .
+
+mudcontent: rdf:type owl:Ontology ;
+    # Description
+    dcterms:title "Multi-User Domain Content ontology" ;
+    dcterms:description """
+    Includes definitions for content in Multi-User Domains
+    """ ;
+    # Ownership
+    dcterms:license <http://creativecommons.org/licenses/by/4.0/> ;
+    dcterms:creator <https://matttennison.inrupt.net/profile/card#me> .
+
+mudcontent:content a rdfs:Class ;
+  rdfs:label "Textual Content"@en ;
+  rdfs:comment "Content describing another model"@en ;
+
+mudcontent:sight a rdfs:Property
+  rdfs:label "Sight"@en ;
+  rdfs:comment "The visual sense"@en ;
+  rdfs:domain mud:content .
```

Comment:
added a missing '.' character here. Every turtle statement ends in a full stop, the semi-colon is a break to say "done with this property, but the next one is about the same subject (`mudcontent:sight`)